diesel/src/query_dsl/locking_dsl.rs

```diff
@@ -30,3 +30,63 @@ where
         self.as_query().for_update()
     }
 }
+
+/// The `skip_locked` method
+///
+/// This trait should not be relied on directly by most apps. Its behavior is
+/// provided by [`QueryDsl`]. However, you may need a where clause on this trait
+/// to call `for_update` from generic code.
```

Comment:
There is a copy-past error in here. This trait is required for `skip_locked` not for `for_update`.